en/config_fw/weight_and_altitude_tuning.md

```diff
@@ -1,52 +1,63 @@
 # Advanced TECS Tuning (Weight and Altitude)
 
-Before following the guidance provided by this section you should have completed the [basic TECS tuning](../config_fw/position_tuning_guide_fixedwing.md#tecs-tuning-altitude-and-airspeed) which established the key performance limitations of the vehicle that are very important for the altitude and airspeed controller to function properly.
-While those limitations have been defined using constant parameters, in reality vehicle performance is not constant and is affected by various factors.
-This section specifically looks at how vehicle weight and air density affect performance and what you can do do compensate for those effects.
+This topic shows how you can compensate for changes to the [weight of the vehicle](#vehicle-weight-compensation) and the [air density](#air-density-compensation), along with information about the [algorithms](#weight-and-density-compensation-algorithms) that are used.
 
-Both the vehicle weight and the air density, which depends on altitude and air temperature, affect vehicle performance.
-For example, when a vehicle climbs to a higher altitude the air density decreases, and usually the maximum climb rate reduces.
-Similarly, the maximum climb rate of a vehicle will reduce with the weight of the vehicle.
+:::warning
+This topic requires that you have already performed [basic TECS tuning](../config_fw/position_tuning_guide_fixedwing.md#tecs-tuning-altitude-and-airspeed).
+:::
 
+[Basic TECS tuning](../config_fw/position_tuning_guide_fixedwing.md#tecs-tuning-altitude-and-airspeed) established the key performance limitations of the vehicle that are required for the altitude and airspeed controller to function properly.
+
+While those limitations are specified using constant parameters, in reality vehicle performance is not constant and is affected by various factors.
 If changes in weight and air density are not taken into account, altitude and airspeed tracking will likely deteriorate in the case where the configuration (air density and weight) deviate significantly from the configuration at which the vehicle was tuned.
 
-In section [Specify the Weight of the Vehicle](../config_fw/weight_and_altitude_tuning.md#specify-the-weight-of-the-vehicle) and [Compensation for Air Density](../config_fw/weight_and_altitude_tuning.md#compensation-for-air-density) you will find the parameters you need to adjust for weight and density compensation to take place.
-In the [Notes](../config_fw/weight_and_altitude_tuning.md#notes-to-the-derivation-of-weight-and-density-compensation) section the curious reader can find more details on theory of each compensation.
+## Vehicle Weight Compensation
```

Comment:
FYI This section is how I would (ideally) like all instructional sections - this is what you need to change and how.